### ISSUE TITLE
If lerna.json is not valid JSON, the file is reported as missing

### ISSUE THREAD

**Issue by edmorley** [lerna/lerna#1067]
If `lerna.json` is not valid JSON, lerna reports the file as missing, rather than saying there was an error parsing it, causing confusion to beginners as to whether they might have used the wrong name for the file, or put it in the wrong place.

## Expected Behavior
```
lerna ERR! EINVALIDLERNA Cannot parse `lerna.json`: <detail message from the loadJsonFile.sync() exception string>
```

## Current Behavior
```
lerna ERR! ENOLERNA `lerna.json` does not exist, have you run `lerna init`?
```

## Possible Solution
Adjust the try-catch here to distinguish the difference between `lerna.json` being missing or invalid:
https://github.com/lerna/lerna/blob/5b8795c952b452e9e2f450673dd2955ca9fe41e6/src/Repository.js#L35-L39
...since contrary to the comment there, the difference does matter IMO.

Or alternatively if that's not desired, the existing error message could be adjusted to say `lerna.json does not exist OR is invalid JSON`.

## Steps to Reproduce (for bugs)
1. Run `yarn add lerna`
2. Create a `lerna.json` with the following content:
```js
{
  "lerna": "2.4.0",
  "version": "1.0.0",
  "packages": [
    "packages/*"
  ],
}
```
3. Run `yarn lerna bootstrap`

## Your Environment
| Executable | Version |
| ---: | :--- |
| `lerna --version` | 2.4.0 |
| `npm --version`  | 5.4.2 |
| `yarn --version` | 1.2.0 |
| `node --version` | 8.6.0 |

| OS | Version |
| --- | --- |
| Windows 10 | 1703 |

**Comment by evocateur** [comment]
This is a bad experience, I agree. I'm tempted to switch to [cosmiconfig](https://github.com/davidtheclark/cosmiconfig#cosmiconfig) and stop worrying about dangling commas...

**Comment by tivac** [comment]
@evocateur That sounds awesome, `cosmicconfig` is great!

**Comment by gaearon** [comment]
You're also going to get this error if you accidentally omit the `"lerna"` key in `lerna.json`.
It should be a version number:

```js
  "lerna": "2.6.0",
```

I omitted it because this is what [Yarn Workspaces blog post](https://yarnpkg.com/blog/2017/08/02/introducing-workspaces/) told me to use:

<img width="953" alt="screen shot 2018-01-11 at 6 47 54 pm" src="https://user-images.githubusercontent.com/810438/34841372-fc384f00-f6ff-11e7-875b-07a1a97dd31f.png">


I'll send a PR to Yarn to correct this.

**Comment by evocateur** [comment]
Published in v2.7.0

**Comment by edmorley** [comment]
Many thanks :-)
